web/src/lib/components/album-page/album-options.svelte

```diff
@@ -45,8 +58,31 @@
       handleError(error, $t('errors.unable_to_save_album'));
     }
   };
+
+  const handleMenuRemove = (user: UserResponseDto): void => {
+    selectedRemoveUser = user;
+  };
+
+  const handleRemoveUser = async (): Promise<void> => {
+    if (!selectedRemoveUser) {
+      return;
+    }
+    try {
+      await removeUserFromAlbum({ id: album.id, userId: selectedRemoveUser.id });
+      onRemove(selectedRemoveUser.id);
+      notificationController.show({
+        type: NotificationType.Info,
+        message: $t('album_user_removed', { values: { user: selectedRemoveUser.name } }),
+      });
+    } catch (error) {
+      handleError(error, $t('errors.unable_to_remove_album_users'));
+    } finally {
+      selectedRemoveUser = null;
+    }
+  };
 </script>
 
+{#if !selectedRemoveUser}
```

Comment:
I have not tested the PR yet. Does it mean when you select a user to remove it, the album-options modal disappear? That does not sound right from a UX perspective.

Reply:
no it did not disappear

Reply:
I tested it and it does disappear. But it would need to refactor the clickOutside actions to change that behavior